app/controllers/demographic_questions_controller.rb

```diff
@@ -60,6 +60,6 @@ def set_demographic_question
 
     # Never trust parameters from the scary internet, only allow the white list through.
     def demographic_question_params
-      params.require(:demographic_question).permit(:text, translations_attributes: [:id, :text, :locale])
+      params.require(:demographic_question).permit(:text, :question_type, translations_attributes: [:id, :text, :locale])
```

Comment:
Hmm, just thinking about the validations, do we want them in here? Maybe we should talk to roger about exposing them to the admins or not

Reply:
Good point @spencerldixon